bot/cogs/events.py

```diff
@@ -0,0 +1,152 @@
+# coding=utf-8
+import logging
+
+from discord import Embed, Member
+from discord.ext.commands import (
+    AutoShardedBot, BadArgument, BotMissingPermissions,
+    CommandError, CommandInvokeError, Context,
+    NoPrivateMessage, UserInputError
+)
+
+from bot.constants import (
+    ADMIN_ROLE, DEVLOG_CHANNEL, DEVOPS_ROLE, MODERATOR_ROLE, OWNER_ROLE, PYTHON_GUILD, SITE_API_KEY, SITE_API_USER_URL
+)
+
+log = logging.getLogger(__name__)
+
+
+class Events:
+    """
+    No commands, just event handlers
+    """
+
+    def __init__(self, bot: AutoShardedBot):
+        self.bot = bot
+
+    async def send_updated_users(self, *users):
+        try:
+            response = await self.bot.http_session.post(
+                url=SITE_API_USER_URL,
+                json=list(users),
+                headers={"X-API-Key": SITE_API_KEY}
+            )
+
+            return await response.json()
+        except Exception as e:
+            log.error(f"Failed to send role updates: {e}")
+            return {}
+
+    async def on_command_error(self, ctx: Context, e: CommandError):
+        command = ctx.command
+        parent = None
+
+        if command is not None:
+            parent = command.parent
+
+        if parent and command:
+            help_command = (self.bot.get_command("help"), parent.name, command.name)
```

Comment:
Including the `(` and `)` is unnecessary, you can accomplish the same feat with simply 
```py
help_command = self.bot.get_command("help"), parent.name, command.name
```